### DISCUSSION TITLE
Access form submission data by index

### DISCUSSION THREAD

**Discussion by scottmetoyer** [statamic/cms#5792] [Q&A]
Probably missing something simple here, but can I access form submission field values by index in an antlers template? Something like this:

```
{{ if success }}
   {{ form:submission in="contact" }}
      {{ data:0 }}
      {{ data:1  }}
   {{ /form:submission }}
{{ /if }}
```

**Comment by edalzell** [comment]
Why, what's your use case?

**Reply by scottmetoyer** [reply]
I've got a form with 10 fields for users to enter answers to a question.

I'd like to loop through the submitted answers to compare them against a collection of items and see if the answers are included in the collection.

Thanks!

**Reply by edalzell** [reply]
If you know the form you're in, then you know the fields? I don't understand why you can't use the fields you know, as you'd want to compare those to named answers, right?

Anyway this **might** work:

```
{{ foreach array="{session:submission}" }}
    ...
{{ /foreach }}
```

**Reply by scottmetoyer** [reply]
I could definitely use the fields directly without a loop, but it felt a little clunky comparing each field against the collection. 

```
{{ if collection | contains:answer_1 }}
 ...
{{ /if }}
{{ if collection | contains:answer_2 }}
 ...
{{ /if }}
...
```

I thought a loop may be a cleaner approach. I'll give your suggestion a shot, thank you!

**Reply by edalzell** [reply]
tbh, I'd likely make a tag that did whatever magic you needed.
